src/Scream.php

```diff
@@ -94,7 +96,7 @@ public function __set($name, $value)
 	 * @param string $name property name
 	 * @throws \Kdyby\StrictObjects\MemberAccessException
 	 */
-	public function __isset($name)
+	public function __isset(string $name): void
```

Comment:
Nette has a `public function __isset(string $name): bool` signature, wouldn't it be better to have it same?

Reply:
But `kdyby/translation` in Nette 3.0 does not work due to this.

```
Declaration of Kdyby\StrictObjects\Scream::__isset(string $name): void must be compatible with Nette\DI\CompilerExtension::__isset(string $name): bool.
```